tests/inputFieldsSpec.js

```diff
@@ -572,8 +572,8 @@
 
                     $rootScope.$digest();
 
-                    expect($(compiledTemplate, 'select[ng-model="hours"] option:nth-child(1)').attr('label')).toBe('00');
-                    expect($(compiledTemplate, 'select[ng-model="hours"] option:nth-child(10)').attr('label')).toBe('09');
+                    expect($(compiledTemplate, 'select[ng-model="hours"] option:nth-child(2)').attr('label')).toBe('00');
```

Comment:
The tests are passing in master, so has something changed in one of the dependencies that causes this?